src/azul/files.py

```diff
@@ -15,3 +16,18 @@ def write_file_atomically(path, mode=0o644):
     except BaseException:
         os.unlink(temp_path)
         raise
+
+
+def file_sha1(path):
+    """
+    >>> file_sha1('/dev/null')
+    'da39a3ee5e6b4b0d3255bfef95601890afd80709'
+    """
+    with open(path, 'rb') as f:
+        sha1 = hashlib.sha1()
+        while True:
+            data = f.read(1024 * 1024)
+            if data == b'':
```

Comment:
```suggestion
            if data:
```

I don't want to live in a world in which .read returned a False-ish value other than empty bytes. In that world, your version would pass that False-ish value to .update and then what?

Reply:
I changed this to `if not data:`